CHANGELOG.md

```diff
@@ -9,20 +9,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Breaking changes
 
 * Ignore the `internal` parameter in packages and `/search` requests. [#765](https://github.com/elastic/package-registry/pull/765)
+* Packages with major version 0 or with prerelease labels are only returned by search requests when they include `prerelease=true` or `experimental=true`. [#785](https://github.com/elastic/package-registry/pull/785)
```

Comment:
Out of curiosity, do you think we should perform an exercise and check how many packages are affected because of this change?

Reply:
Packages with prerelease tags: 1, apm-server 8.0.0-rc1
Packages with major version 0: [48](https://gist.github.com/jsoriano/e0750af2d3f25d037cb3fa2a87ae0b1d), they are all marked as beta or experimental, except a couple of them. We will have to think what to do about them. The couple of ga ones should be probably versioned as 1.x.

Btw, I have also found that kibana, logstash and microsoft packages are marked as experimental, but have 1.x versions, we will have to check this too.

I can delay the merge of this till we clarify the status of these packages.